src/fiat-connect-client.ts

```diff
@@ -82,9 +83,22 @@ export class FiatConnectClient implements FiatConnectApiClient {
    * @returns a Promise resolving to the literal string 'success' on a
    * successful login or an Error response.
    */
-  async login(): Promise<Result<'success', ResponseError>> {
+  async login(params?: LoginParams): Promise<Result<'success', ResponseError>> {
```

Comment:
update jsdoc to include params? Although not sure if we've been doing this consistently